.gitignore

```diff
@@ -10,4 +10,5 @@ buildNumber.properties
 # https://github.com/takari/maven-wrapper#usage-without-binary-jar
 .mvn/wrapper/maven-wrapper.jar
 .idea/
-Java.iml
\ No newline at end of file
+Java.iml
```

Comment:
just remove file Java.iml if it is in git index
and add all .iml files to gitignore with wildcards: **/*.iml